services/api/service.go

```diff
@@ -1180,6 +1182,25 @@ func (api *RelayAPI) handleGetHeader(w http.ResponseWriter, req *http.Request) {
 	}
 
 	log.Debug("getHeader request received")
+	defer func() {
+		s, err := strconv.ParseInt(slotStr, 10, 64)
+		if err != nil {
+			log.WithError(err).Error("could not parse slot to int64")
+			return
+		}
+		metrics.GetHeaderLatencyHistogram.Record(
+			req.Context(),
+			float64(time.Since(requestTime).Milliseconds()),
+		)
+		metrics.GetHeaderCount.Add(
+			req.Context(),
+			1,
+			otelapi.WithAttributes(
+				attribute.Int64("slot", s),
+				attribute.Int64("floorSecIntoSlot", msIntoSlot/1000),
+			),
```

Comment:
**remark:**

not so relevant if we just drop this counter, but generally this approach is not desirable as each new combination of labels (`slot` and `floorSecIntoSlot` in this case) will cause prometheus to start a fresh counter (which will eventually explode as the size of the scrape payload will keep growing with time).

Reply:
how do we label metrics not using attributes then?